src/main/js/components/rows/Row.jsx

```diff
@@ -93,14 +91,14 @@ class Row extends React.Component {
       // We want to see single-language value even if not expanded
       if (!cell.isMultiLanguage && !self.props.isRowExpanded) {
         return <Cell key={idx} cell={cell} langtag={langtag} selected={selected} editing={editing}
-                     shouldFocus={shouldFocus} />;
+                     shouldFocus={shouldFocus} row={self.props.row} table={self.props.table} />;
```

Comment:
If the `map` call in line 72 would use an arrow function (`(cell, idx) => { ... }`), you could get rid of the `self` and just use `this` as you can in most other programming languages... :)